src-vue/src/views/DeveloperView.vue

```diff
@@ -43,6 +47,24 @@ export default defineComponent({
                 position: 'bottom-right'
             });
         },
+        async checkLinuxCompatability() {
+            let LinuxCompatabile = await invoke("linux_checks");
+            if (!LinuxCompatabile) {
+                ElNotification({
+                title: 'Not linux compatabile',
+                message: 'GLIBC is not version 2.33 or greater',
+                type: 'error',
+                position: 'bottom-right'
+                });
+            } else {
+                ElNotification({
+                title: 'Linux compatabile',
+                message: 'No error reported',
+                type: 'succes',
```

Comment:
Pretty sure it's speleed:

```suggestion
                type: 'success',
```

Reply:
yeah that ones my bad lol

Reply:
Yup https://github.com/GeckoEidechse/FlightCore/actions/runs/3200168318/jobs/5226860536#step:6:32

Reply:
succes